### DISCUSSION TITLE
Searching for a motif of changing lengths

### DISCUSSION THREAD

**Discussion by royyanovski** [TDAmeritrade/stumpy#446] [Q&A]
Hi, I am using stumpy for a motif search. I already have the pattern/subsequence I am looking for but I want to be able to search for this pattern as it appears in different lengths (i.e. stretched or shrunk). What would be the most efficient way to do this? and how can I control the range of acceptable lengths for the motif?

Any ideas and suggestions will be great.
Thanks.

**Comment by JaKasb** [comment]
Dynamic Time Warping
Max allowed length = warping constraint

**Reply by royyanovski** [reply]
Thanks. I understand that this is not something that can be done via stumpy. Is this true? Is there another python package that does that?

**Reply by seanlaw** [reply]
That's correct. STUMPY does not offer any dynamic time warping capabilities and, unfortunately, we are unable to endorse any particular DTW package.
